orttraining/orttraining/core/session/training_session.h

```diff
@@ -426,7 +426,11 @@ class TrainingSession : public InferenceSession {
                                       bool use_fp16_initializer,
                                       std::unordered_map<std::string, NodeArg*>& fp32_weight_name_to_fp16_node_arg);
 
-  std::unordered_set<std::string> GetTrainableModelInitializers(const ImmutableWeights& immutable_weights) const;
+  /** Discover all trainable initializers by reverse DFS from output
+  @param immutable_weights do not include initializers matching an (op_type, input_index, value) entry from this table
+  @param backprop_source_name reverse DFS back propagation source name (i.e. loss name or pipeline send output name)
+  */
+  std::unordered_set<std::string> GetTrainableModelInitializers(const ImmutableWeights& immutable_weights, const std::string& backprop_source_name) const;
```

Comment:
>std::unordered_set<std::string> GetTrainableModelInitializers(const ImmutableWeights& immutable_weights, const std::string& backprop_source_name) const; [](start = 2, length = 152)

nit: maximum line length is 120 (FYI onnx coding style: [https://github.com/microsoft/onnxruntime/blob/5e0928a777745a1536332973418dbbb7ccbf9d52/docs/Coding_Conventions_and_Standards.md](https://github.com/microsoft/onnxruntime/blob/5e0928a777745a1536332973418dbbb7ccbf9d52/docs/Coding_Conventions_and_Standards.md))